hardware_interface/src/resource_manager.cpp

```diff
@@ -73,10 +76,16 @@ class ResourceStorage
   static constexpr const char * system_interface_name = "hardware_interface::SystemInterface";
 
 public:
-  ResourceStorage()
+  // TODO(VX792): Change this when HW ifs get their own update rate,
```

Comment:
Yes, the async HW itself. Since when HW ifs get their own update rate, then this could be parsed from the config file directly to the HardwareInfo struct of the component.